xray/analyze.py

```diff
@@ -84,6 +86,30 @@ def __init__(self, csv_name, pcap_name, count, test_type):
             fn(ax[i, 1])
 
         plt.show()
+        if save_output:
+            plt.savefig(test_path + ".png")
```

Comment:
Have you tried running this locally with passing `--ascii`? I think I read somewhere that `savefig` needs to be called before `show` because `show` will clear the plot, leaving nothing for `savefig` to save. I saw in the artifacts that it's working though, which is weird. But maybe move the `if save_output` to be before the `show`

Reply:
hmm yeah maybe because it doesn't show the UI on the CI.

Reply:
Swapped the order